cfme/tests/middleware/deployment_methods.py

```diff
@@ -18,8 +18,8 @@
 
 DEPLOYMENT_URL = 'http://{}:8180/{}'
 WAR_EXT = '.war'
-DELAY = 10
-NUM_SECS = 120
+DELAY = 30
```

Comment:
**Required**
Keep the delay down to 10 seconds unless there is a strong reason to not query more often in `wait_for()`.

Reply:
I set it to query less often (30 secs) and increased the timeout to 10 minutes. This is done because when deploying a lot of archives on the server, it starts inventorying the deployments slower, and tests were failing.

Reply:
I think the performance issue is not in CFME/MM side, but on EAP side itself